spec/auth_test_helper.rb

```diff
@@ -0,0 +1,26 @@
+    def stub_auth
```

Comment:
There's some indentation here that it would be good to clear up. I think this should be wrapped in a Ruby module or ideally we could use RSpec's [shared contexts](https://relishapp.com/rspec/rspec-core/v/3-8/docs/example-groups/shared-context).

This can be done in a later PR though.